### ISSUE TITLE
Add sort functionality for poll lists

### ISSUE THREAD

**Issue by markusphil** [OpenTechSchool-Leipzig/poll_app#30]
I already implemented sorting options for the questionlist and created some helperfunctions. This could be easily adopted for the poll / template list.

But maybe you have a better idea for the sorting functionality =)

**Comment by AnBusse** [comment]
Hi, I'd like to give this issue a shot

**Comment by markusphil** [comment]
Of course! just go ahead and let me know if you encounter any problems :blush:

**Comment by AnBusse** [comment]
@markusphil so I have the app up and running locally. I'd like to get the requirements right:
I see a basic filtering is implemented on the poll list already (see screenshot attached). You mention a question list that has the desired filtering already, but while I can see the view file `QuestionList.vue` I can't seem to find it used in the application right now, is that correct?
![Screenshot_2019-10-06 poll-app](https://user-images.githubusercontent.com/22819829/66275015-dfd33a00-e884-11e9-8f08-a11c29221ec8.png)

**Comment by markusphil** [comment]
I tried to generalize the "FilterBar" the filterbar so it can be used with all available lists. The sort and filter logic itself is contained in the list components.  The `QuestionList.vue` file is one of this components and it is used on the `CreatePoll.vue` View (it's the list on the left side).
